### PR TITLE
Upgrade Validation Library

### PR DESCRIPTION
**Pull request by fkpwolf**
[AB#397298](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/397298)

### CODE REVIEW

File: src/docfx/docfx.csproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Identity" Version="1.3.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.1.0" />
-    <PackageReference Include="docs.validation" Version="1.0.0-beta-00559-25e9ad1" />
```
Comment (OsmondJiang) [inline src/docfx/docfx.csproj:17]: could you add test for your change?

Comment (fkpwolf) [reply src/docfx/docfx.csproj:17]: added

**Review by yufeih — APPROVED** [review]

**Review by OsmondJiang — APPROVED** [review]

**Comment by OsmondJiang** [comment]
@fkpwolf could you confirm the diffs are expected? Seems below ones are not expected:

image

**Comment by fkpwolf** [comment]
> @fkpwolf could you confirm the diffs are expected? Seems below ones are not expected:
> 
> image

I check log in DevOps, it has error like
```
2021-05-06T09:19:21.1617924Z pivot-id-unused ./test/docs/zones/zones-isolated.md(0,0): Pivot ID 'android' is defined in 'platform' group in 'zone-pivot-groups.yml', but not used. You must use all pivot IDs in a pivot group to avoid blank tabs.
2021-05-06T09:19:21.1618845Z pivot-id-unused ./test/docs/zones/zones-isolated.md(0,0): Pivot ID 'linux' is defined in 'platform' group in 'zone-pivot-groups.yml', but not used. You must use all pivot IDs in a pivot group to avoid blank tabs.
```
rerun the pipeline.

**Comment by mingwli** [comment]
The failure spec test is tracked in this task **AB#417016**. It may has to retry the build before the bug get fixed Sorry for the inconvenience. @fkpwolf

**Pull request merged**